projects/fsastorefrontlib/src/core/my-account/store/reducers/quote.reducer.ts

```diff
@@ -13,7 +13,7 @@ export function reducer(
 ): QuoteState {
   switch (action.type) {
     case fromAction.LOAD_QUOTES_SUCCESS: {
-      const quotes = [...action.payload];
+      const quotes = action.payload ? [...action.payload] : [];
```

Comment:
What do you think about handling this in effect instead? Maybe we should try to keep reducers logic to a minimum, as it was intended? Just returning new state.
Would it be better to add this to effect instead:
```
          if (quotes) {
            return new fromActions.LoadQuotesSuccess(quotes);
          } else {
            return new fromActions.LoadQuotesSuccess([]);
          }
```